views/index.jade

```diff
@@ -21,3 +21,16 @@ block content
                             p.itinerary-id(hidden)=itinerary.id
                             .text-right.col-xs-2
                                 button.itinerary-delete.glyphicon.glyphicon-trash.btn.btn-primary(type="Submit")
+                              script.
```

Comment:
This script does not appear to change what is on the index page

Reply:
the fix is not on index.jade but on route-script.js

Reply:
just tested on local server, and it works. Going to say it looks good on the review